src/commands/remote_signer_setup.py

```diff
@@ -220,3 +268,115 @@ async def main(remove_existing_keys: bool) -> None:
         f' Successfully configured operator to use remote signer'
         f' for {len(keystores)} public key(s)!',
     )
+
+
+# pylint: disable-next=too-many-locals
+def _update_db(
+    db_url: str | None,
+    db_encrypt_key: str | None,
+    key_shares: dict | None,
+) -> None:
+    check_db_connection(db_url)
+
+    with open(settings.keystores_password_file, encoding='utf-8') as f:
+        keystores_password = f.read().strip()
+
+    private_keys = []
+
+    with click.progressbar(
+        glob.glob(os.path.join(settings.keystores_dir, 'keystore-*.json')),
+        label='Loading keystores...\t\t',
+        show_percent=False,
+        show_pos=True,
+    ) as _keystore_files:
+        for filename in _keystore_files:
+            try:
+                keystore = ScryptKeystore.from_file(filename).decrypt(keystores_password)
+                private_keys.append(int.from_bytes(keystore, 'big'))
+            except (json.JSONDecodeError, KeyError) as e:
+                click.secho(
+                    f'Failed to load keystore {filename}. Error: {str(e)}.',
+                    fg='red',
+                )
+
+    if key_shares is None or 'keystores' not in key_shares or 'passwords' not in key_shares:
+        raise ValueError('key_shares is malformed or None')
+
+    with click.progressbar(
+        length=len(key_shares['keystores']),
+        label='Loading key shares...\t\t',
+        show_percent=False,
+        show_pos=True,
+    ) as progressbar:
+        for index, key_share_str in enumerate(key_shares['keystores']):
+            try:
+                keystore_password = key_shares['passwords'][index]
+                # Parse key_share from string to dictionary
+                key_share = json.loads(key_share_str)
+                keystore = ScryptKeystore.from_json(key_share).decrypt(keystore_password)
+                private_keys.append(int.from_bytes(keystore, 'big'))
+            except (json.JSONDecodeError, KeyError) as e:
+                click.secho(
+                    f'Failed to load keystore {key_share_str}. Error: {str(e)}.',
+                    fg='red',
+                )
+            progressbar.update(1)
+
+    database = Database(
+        db_url=str(db_url),
+    )
+    encryptor = Encryptor(db_encrypt_key)
+
+    database_records = _encrypt_private_keys(
+        private_keys=private_keys,
+        encryptor=encryptor,
+    )
+    database.upload_keys(keys=database_records)
+    total_keys_count = database.fetch_public_keys_count()
+
+    configs = [
+        _read_config_file_to_record(
+            settings.remote_signer_config_file, 'remote_signer_config.json'
```

Comment:
Why do we need record name? We can use ID=`0` for remote signer and ID=`1` for deposit data